src/wp-includes/default-filters.php

```diff
@@ -603,6 +603,14 @@
 // Global styles custom CSS.
 add_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles_custom_css' );
 
+// Block template.
+add_filter( 'the_block_template_html', 'shortcode_unautop', 7 );
+add_filter( 'the_block_template_html', 'do_shortcode', 8 );
+add_filter( 'the_block_template_html', 'do_block_template_blocks', 9 );
+add_filter( 'the_block_template_html', 'wptexturize' );
+add_filter( 'the_block_template_html', 'wp_filter_content_tags' );
+add_filter( 'the_block_template_html', 'convert_smilies', 20 );
```

Comment:
In the original code that is being changed here, `convert_smilies()` was run _before_ `wp_filter_content_tags()`. While I'm not opposed to use the same priority as `convert_smilies` uses elsewhere, we should make sure using a different order here wasn't an intentional decision (and there is a good chance it wasn't).

But if we don't feel strongly, it would be safer to leave out the priority on `convert_smilies()` so that the original order is maintained.
```suggestion
add_filter( 'the_block_template_html', 'wp_filter_content_tags', 12 );
add_filter( 'the_block_template_html', 'convert_smilies' );
```